docker/deployed/staging/api/config.json

```diff
@@ -7,7 +7,8 @@
     "TLSKey": "${VALIDATOR_TLS_KEY}"
   },
   "Gateway": {
-    "ExternalURIPrefix": "https://staging.tableland.network"
+    "ExternalURIPrefix": "https://staging.tableland.network",
+    "MetadataRendererURI": "https://render.tableland.xyz"
```

Comment:
Shouldn't this URL be empty for staging? Because I think it would point out to testnet tables. 

If empty, prob we need to adjust the `fmt.Sprintf(...)` below to cover that case. Or see if we can make another deployment of  `https://render.tableland.xyz` for the staging env.